README.md

```diff
@@ -3,13 +3,16 @@
 [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/RubyData/docker-stacks/master?filepath=README.ipynb)
 
 RubyData Docker stacks are a set of ready-to-run Docker images containing Jupyter and data tools for Rubyists.
+This repository is based on [jupyter/docker-stacks](https://github.com/jupyter/docker-stacks).
 
 ## Quick Start
 
 You can try the recent build of the RubyData/minimal-notebook image on mybinder.org by click the link below:
 
 https://mybinder.org/v2/gh/RubyData/docker-stacks/master?filepath=README.ipynb
 
+If you [have Docker installed](https://docs.docker.com/install/) know [which Docker image](how_to_select.md) you want to use, and want to launch a single Jupyter Notebook server in a container.
```

Comment:
This sentence seems not correct English (or my English skill is not enough to recognize it).
Could you please change this sentence to the following simpler English?

```
Otherwise, you can launch a single Jupyter Notebook server in a Docker container.  Two Docker images are available. You can know which is suitable for you in [this document](how_to_select.md).
```

Reply:
Thank you for your suggestion. I applied it.